Scope.Client/Loader/BepInExLoader.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="BepInExLoader.cs" company="Scope SL">
+// Copyright (c) Scope SL. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Scope.Client.Loader
+{
+    using BepInEx;
+    using BepInEx.IL2CPP;
+    using HarmonyLib;
+    using Scope.Client.Loader;
+
+    /// <summary>
+    /// Used to load the BepInEx.
+    /// </summary>
+    [BepInPlugin("scopeclient.client.github", "ScopeClient", "0.0.1")]
+    [BepInProcess("SCPSL.exe")]
+    public class BepInExLoader : BasePlugin
+    {
+        private Client _client;
+
+        /// <summary>
+        /// Gets the <see cref="Harmony"/> instance.
+        /// </summary>
+        public static Harmony Harmony { get; private set; }
+
+        /// <summary>
+        /// Gets the <see cref="BepInExLoader"/> instance.
+        /// </summary>
+        public static BepInExLoader Instance { get; private set; }
+
+        /// <summary>
+        /// Gets the <see cref="Client"></see>.
+        /// </summary>
+        public Client Client => _client ?? (_client = new Client());
+
+        /// <inheritdoc/>
+        public override void Load()
+        {
+            Instance = this;
+            Harmony = new Harmony("scopeclient.client.github");
```

Comment:
I'd maybe use some sort of UUID like datetime.ticks for this but it's not really necessary.

Reply:
as like $"scopeclient.client.github.{DateTime.Now.Ticks}"

Reply:
It's pretty tricky, and it'd not add anything imo.